context/ip_allocator.go

```diff
@@ -95,13 +91,12 @@ func (a *IPAllocator) Allocate(imsi string) (net.IP, error) {
 		return nil, errors.New("ip allocation failed" + err.Error())
 	} else {
 		// smfCount := MongoDBLibrary.GetSmfCountFromDb()
-
-		smfCountStr := os.Getenv("SMF_COUNT")
-		smfCount, _ := strconv.Atoi(smfCountStr)
-		ip := IPAddrWithOffset(a.ipNetwork.IP, int(offset)+(int(smfCount)-1)*5000)
```

Comment:
This looks important if there are multiple SMF deployed, can this code be kept and still have the tests passing?

Reply:
We made the changes in the tests, not in this method in the final patch.